packages/vite-plugin-svelte/src/utils/id.js

```diff
@@ -137,6 +137,17 @@ function normalize(filename, normalizedRoot) {
 	return stripRoot(normalizePath(filename), normalizedRoot);
 }
 
+/**
+ * create a normalizer function for root
+ *
+ * @param {string} root
+ * @returns {(filename: string) => string}
+ */
+export function buildFilenameNormalizer(root) {
+	const normalizedRoot = normalizePath(root);
```

Comment:
I think we can skip the normalizePath here since Vite already normalizes it. We pass the root from https://github.com/sveltejs/vite-plugin-svelte/blob/bd915c1317127a716a22cdd8c44d66d766e0f513/packages/vite-plugin-svelte/src/utils/options.js#L212